lib/modem_info/Kconfig

```diff
@@ -37,6 +37,27 @@ config MODEM_INFO_THREAD_PRIO
 	default 0 if !MULTITHREADING
 	default 9
 
+config MODEM_INFO_DEVICE_STRING_NETWORK
+	bool
+	default y
+	help
+		Add the network information to the returned
+		device string.
+
+config MODEM_INFO_DEVICE_STRING_SIM
+	bool
+	default y
+	help
+		Add the sim information to the returned
```

Comment:
Add the SIM card information ...

Reply:
Note that when there's no prompt in the configs (nothing after `bool`), these settings can only be changed by other symbols, and not using menuconfig or project config files.